views/tables/AdminUsersTable/AdminUsersTable.test.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+import { render, screen } from '@testing-library/react'
+import { LocalizationProvider } from '@mui/x-date-pickers'
+import { AdapterMoment } from '@mui/x-date-pickers/AdapterMoment'
+
+import AdminUsersTable from '.'
+import { createUser } from '../../../test/fixtures'
+
+describe('AdminUsersTable', () => {
+  const users = [
+    createUser({
+      uid: 'user',
+      mail: 'test@example.com',
+      role: 'admin',
+    }),
+    createUser({
+      uid: 'foo',
+      mail: 'foo@example.com',
+      role: 'member',
+    }),
+  ]
+  it('renders the table', () => {
+    render(
+      <LocalizationProvider dateAdapter={AdapterMoment}>
```

Comment:
```suggestion
      <LocalizationProvider dateAdapter={AdapterMoment}>
```
Should we add this provider to the page renderer export in the tests directory..?

Reply:
a2f3b1f